### ISSUE TITLE
ORG_ID policy replacement variable does not get substituted in SCP

### ISSUE THREAD

**Issue by zakbatinica-reply** [awslabs/landing-zone-accelerator-on-aws#161]
**Describe the bug**

When creating an SCP using the `${ORG_ID}` variable referenced in the [documentation](https://docs.aws.amazon.com/solutions/latest/landing-zone-accelerator-on-aws/working-with-solution-specific-variables.html) the variable is not substituted with the organisation ID, instead the literal string `${ORG_ID}` is used.

**To Reproduce**

Create an SCP using the following configuration:

organization-config.yaml
```yaml
serviceControlPolicies:
  - name: ResourcePerimeter
    description: >
      Only allow interaction with resources from within our AWS organization.
    policy: service-control-policies/resource-perimeter.json
    type: customerManaged
    deploymentTargets:
      organizationalUnits:
        - Root
```

service-control-policies/resource-perimeter.json

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "EnforceResourcePerimeter",
            "Effect": "Deny",
            "Action": "*",
            "Resource": "*",
            "Condition": {
                "StringNotEqualsIfExists": {
                    "aws:ResourceOrgID": "${ORG_ID}"
                }
            }
        }
    ]
}
```

**Expected behavior**

Assuming the organization ID was `o-1234` I expected the final SCP would render like this:

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "EnforceResourcePerimeter",
            "Effect": "Deny",
            "Action": "*",
            "Resource": "*",
            "Condition": {
                "StringNotEqualsIfExists": {
                    "aws:ResourceOrgID": "o-1234"
                }
            }
        }
    ]
}
```

**Please complete the following information about the solution:**

- Version: v1.3.2
- Region: eu-west-2
- Was the solution modified from the version published on this repository? No
- If the answer to the previous question was yes, are the changes available on GitHub? N/A
- Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the sevices this solution uses? No
- Were there any errors in the CloudWatch Logs? No

**Screenshots**

Resulting SCP from above configuration:

![Screenshot 2023-05-30 at 14 00 05](https://github.com/awslabs/landing-zone-accelerator-on-aws/assets/127220646/7f8d3b22-373c-496b-931e-41284889998c)

**Additional context**

I think this is a bug, but if it's a case that policy replacement variables can't be used in SCP's then I think the [documentation](https://docs.aws.amazon.com/solutions/latest/landing-zone-accelerator-on-aws/working-with-solution-specific-variables.html) should be updated to clarify which policy documents are supported.

**Comment by awsclemj** [comment]
Hello @zakbatinica-reply, and thank you for reporting this issue!

I can confirm that our team has replicated this issue and we are currently working on the fix. The fix is targeted to be included in our next release. I will leave this issue open for now and we'll update you once it's been incorporated into a release branch.

Thank you for your interest and support of the solution!

**Comment by erwaxler** [comment]
Hi @zakbatinica-reply , thank you for your patience on this issue. This behavior was fixed in our latest release v1.4.2, please update your LZA deployment to version 1.4.2 in order to resolve the behavior. I'll be closing this issue now that the fix has been released, thank you again for bringing this to our attention.
